javascript-injectors/README.md

```diff
@@ -1,18 +1,20 @@
-# Website Automation (aka JavaScript Injector)
+# Site Automation (aka JavaScript Injector)
 
 **NOTE: This feature is not yet exposed to customers**
 
 Screenly's JavaScript Injector feature allows users to perform automation on websites, such as:
 
-* Logging into websites using credentials (or a cookie)
+* Logging into sites using credentials (or a cookie)
 * Close modals, such as GDPR consent dialogues
 * Scroll down a page
 
 ## Usage
 
-To use the JavaScript Injector, you need first to create the web asset. You can do that in the [user interface](https://login.screenlyapp.com) or [using the API](https://developer.screenlyapp.com/#operation/assets_create). (While you could include the JavaScript directly in the asset creation, it can be beneficial to do the JavaScript snippet as a PATCH as it allows you to update your JavaScript code easily.)
```

Comment:
Why was this removed?

Reply:
Not removed actually. Just a newline to fix a lint issue saying a very long line with some 412 characters.